aksel.nav.no/website/pages/templates/skjemavalidering/demo.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect } from "react";
+import { ArrowLeftIcon, ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Button,
+  ErrorSummary,
+  HGrid,
+  Heading,
+  Page,
+  Radio,
+  RadioGroup,
+  TextField,
+  VStack,
+} from "@navikt/ds-react";
+
+const validatePersonnummer = (p: string) => {
+  if (p.length === 0) {
+    return "Du må fylle ut personnummer.";
+  }
+  if (!/^\d{11}$/.test(p)) {
+    return "Personnummer må være 11 siffer.";
+  }
+  return "";
+};
+
+const Example = () => {
+  const errorSummaryRef = React.useRef<HTMLDivElement>(null);
+
+  const [hasTriedToSubmit, setHasTriedToSubmit] = React.useState(0);
+  const [formSubmitted, setFormSubmitted] = React.useState(false);
+
+  const [personnummer, setPersonnummer] = React.useState("");
+  const [personnummerError, setPersonnummerError] = React.useState("");
+  const [transportmiddel, setTransportmiddel] = React.useState("");
+  const [transportmiddelError, setTransportmiddelError] = React.useState(false);
+
+  function onSubmit(event: React.FormEvent) {
+    event.preventDefault();
+    setHasTriedToSubmit(hasTriedToSubmit + 1);
+    let isValidForm = true;
+
+    const personnummerErrorMsg = validatePersonnummer(personnummer);
+    setPersonnummerError(personnummerErrorMsg);
+    if (personnummerErrorMsg) {
+      isValidForm = false;
+    }
+    const transportmiddelInvalid = !transportmiddel;
+    setTransportmiddelError(transportmiddelInvalid);
+    if (transportmiddelInvalid) {
+      isValidForm = false;
+    }
+
+    if (isValidForm) {
+      setFormSubmitted(true);
+    }
+  }
+
+  useEffect(() => {
+    errorSummaryRef.current?.focus();
+  }, [hasTriedToSubmit]);
+
+  const showErrorSummary = Boolean(
+    hasTriedToSubmit && (personnummerError || transportmiddelError),
+  );
+
+  if (formSubmitted)
+    return (
+      <Page style={{ marginTop: 20 }}>
+        <Page.Block as="main" width="lg" gutters>
+          <VStack gap="8" align="center">
+            <Heading size="large">Demo slutt</Heading>
+            <Button
+              onClick={() => {
+                location.hash = "";
+                location.reload();
+              }}
+            >
+              Nullstill
+            </Button>
+          </VStack>
+        </Page.Block>
+      </Page>
+    );
+
+  return (
+    <Page style={{ marginTop: 20 }}>
```

Comment:
```suggestion
<Box marginBlock="5 32">
    <Page style={{ marginTop: 20 }}>
```

Reply:
If using "5 32" here, one can remove paddingBlock from `<HGrid` below (comment)

Reply:
Removed the margin